joern-cli/frontends/csharpsrc2cpg/src/main/scala/io/joern/csharpsrc2cpg/TypeMap.scala

```diff
@@ -145,7 +164,7 @@ class TypeMap(astGenResult: AstGenRunnerResult, initialMappings: List[NamespaceT
       .json(ParserKeys.Variables)
       .arr
       .map(AstCreatorHelper.createDotNetNodeInfo(_))
-      .map(AstCreatorHelper.nameFromNode)
+      .map(fieldDecl => s"${AstCreatorHelper.nameFromNode(fieldDecl)}")
```

Comment:
This doesn't appear to do anything meaningful so I'd change it back
```suggestion
      .map(AstCreatorHelper.nameFromNode)
```